docker/grpc.Dockerfile

```diff
@@ -40,11 +43,19 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
         checkinstall \
         zlib1g-dev \
         doxygen \
-        graphviz
+        graphviz \
+        docker-compose
 
 RUN pip3 install behave PyHamcrest pathlib2
 RUN apt-get clean
 
+# Install Docker Enginer
```

Comment:
why install docker in the container?

Reply:
Got it, but shouldn't those be installed on another image that extends the build image?